omod/src/main/resources/findPatient.gsp

```diff
@@ -0,0 +1,34 @@
+<%
+    ui.decorateWith("appui", "standardEmrPage")
+%>
+<script type="text/javascript">
+<% if (breadcrumbs) { %>
+    var breadcrumbs = ${ breadcrumbs };
+<% } else { %>
+    var breadcrumbs = [
+        { icon: "icon-home", link: '/' + OPENMRS_CONTEXT_PATH + '/index.htm' },
+        { label: "${ ui.message(label)}"}
+    ];
+<% } %>
+
+    jq(function() {
```

Comment:
@ibacher I think that we might not need a duplicate this page.

What's your say? Also I am not sure that if I have missed out some configuration

Reply:
I *think* we need this to keep the connection between the controller and the page, although, I would note that this should actually we be in `omod/src/main/webapp/pages`, which I think will ensure it's in the right place.